src/Extensions/HeaderButtonExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SilverStripe\StartupTheme;
+
+use SilverStripe\Forms\FieldList;
+use SilverStripe\LinkField\Form\LinkField;
+use SilverStripe\LinkField\Models\Link;
+use SilverStripe\ORM\DataExtension;
+
+class HeaderButtonExtension extends DataExtension {
+
+    private static array $has_one = [
+        'HeaderLink' => Link::class
+    ];
+
```

Comment:
I could not publish the link without adding an $owns array - please add 

private static array $owns = ['HeaderButton'];

Reply:
yes, that is why it is in draft mode. 
I think owns the `HeaderLink`